site/app/templates/grading/electronic/AutogradingPanel.twig

```diff
@@ -12,6 +12,9 @@
                 </button>
                 <div class="dropdown-menu">
                     <button id="autograding-results-regrade-all" class="dropdown-item">Regrade All Versions</button>
+                    {% for version in versions %}
+                    <button  id={{loop.index}} class="dropdown-item">Version #{{ loop.index }} Score: {{ version.points }} / {{ total_points }} {%- if loop.index == active_version -%} &nbsp;&nbsp;&nbsp;GRADE THIS VERSION {%- endif -%}</button>
```

Comment:
The button here (and all buttons in this file) should have the `type="button"` attribute on them.

The id attribute of these buttons should be changed to something more specific (i.e. `autograding-results-regrade-{id}`).

The dropdown menu also currently sticks on the panel when you have it open and scroll past the `Regrade Active Version` button - while this isn't an issue itself, it becomes a problem when trying to regrade assignments that students have submitted many times (15 on my computer, possibly less on mobile) - the menu becomes taller than the panel and anything outside of the panel height cannot be clicked on, even if you scroll down since the dropdown menu is static on the panel. One way to solve this could be to make it so the dropdown menu is no longer sticky on the panel.